README.md

```diff
@@ -1,9 +1,7 @@
 # Distro plugin for [Tutor](https://docs.tutor.overhang.io)
 
-
-This plugin is a tool to facilitate the customization of an Openedx instance, adding commands and settings to have an easy-to-use
-and a ready-to-deploy in local or in development openedx distribution.
-
+This plugin is a tool to facilitate the customization of an Open edX instance, adding commands and settings to have an easy-to-use
```

Comment:
Here's a re-wording suggestion, but it needs to be reviewed cause I'm not sure whether I missed something:

This plugin facilitates customizing Open edX by adding commands and settings to make your instance easy to use and deploy locally or in development environments.